server/services/MailerService.ts

```diff
@@ -0,0 +1,75 @@
+import nodemailer, { Transporter, SentMessageInfo } from 'nodemailer';
+import { IMailerService } from 'types/services';
+import Utilities from 'server/util/Utilities';
+
+// @todo add ourEmail, emailUsername, emailPassword, and emailService as
+// environment variables when they become available. Temporary placeholders
+// provided until updated info available.
+export default class MailerService implements IMailerService {
+  emailList: Array<string>;
+  subject: string;
+  htmlEmail: string;
+  backupText: string;
+  transporter: Transporter;
+  ourEmail: string;
+  emailUsername: string;
+  emailPassword: string;
+  emailService: string;
+
+  constructor(
+    emailList: Array<string>,
+    subject: string,
+    htmlEmail: string,
+    backupText?: string,
+  ) {
+    this.emailList = emailList;
+    this.subject = subject;
+    this.htmlEmail = htmlEmail;
+    this.backupText = backupText || '';
+
+    // to be replaced with env vars
+    this.ourEmail = 'ourEmail@placeholder.place';
```

Comment:
can you please replace this placeholder with env, and define this in `env.sample`.

Reply:
Also, please update the readme, to copy env from `env.sample`

Reply:
Added :)